test/snowflex/worker_test.exs

```diff
@@ -34,8 +34,10 @@ defmodule Snowflex.WorkerTest do
     end
 
     test "does not send a heartbeat if `keep_alive?` is false" do
-      start_supervised!({Worker, @without_keep_alive})
-      Process.sleep(15)
+      capture_log(fn ->
+        start_supervised!({Worker, @without_keep_alive})
+        Process.sleep(15)
+      end)
```

Comment:
I prefer [the way we do this in dpi](https://github.com/pepsico-ecommerce/dpi/blob/main/config/test.exs#L15-L19) where we configure test to not write info and debug to console